js_task1/firstTask.js

```diff
@@ -0,0 +1,26 @@
+const input = document.querySelector(".first-task input");
```

Comment:
You can write whole block like: 

const input = document.querySelector(".first-task input"),
      pushButton = document.querySelector(".first-task button"),
      outArea = document.querySelector(".first-task .out-area"),
      symbol = " $ "; // output symbol

Reply:
changed